File: 172-FactorialTrailingZeroes/172-FactorialTrailingZeroes.py
# Last updated: 7/30/2025, 12:24:32 PM
class Solution:
    def trailingZeroes(self, n: int) -> int:
        count = 0
        while(n>=5):
            n //= 5
            count += n
        return count 
                         

        

File: 172-FactorialTrailingZeroes/172-FactorialTrailingZeroes.cpp
// Last updated: 7/30/2025, 12:24:28 PM
class Solution {
public:
    int trailingZeroes(int n) {
        int c = 0,temp = n;
        while (temp>4){
            temp/=5;
            c+=temp;
        }
        return c;
    }
};